examples/README.md

```diff
@@ -0,0 +1,7 @@
+# cuDF Examples
+This directory contains examples of cuDF usage. Some examples are simple while others are more involved. Each example is held within a subdirectory. Each example will have its own README to explain intended use, assist with building, and running.
```

Comment:
There's only one example, so how can you say some are simple while others are more involved?

Reply:
You are right. I was thinking ahead about other examples I had planned to add. I will reword it.

Reply:
Actually I'll just remove that part for now.